app/src/main/java/nl/tudelft/bbw/controller/KeyReader.java

```diff
@@ -69,9 +70,9 @@ private static void initialize(String filePath) throws IOException {
      *
      * @return private key
      */
-    public static EdDSAPrivateKey readPrivateKey() throws InvalidKeySpecException, IOException {
+    public static EdDSAPrivateKey readPrivateKey(String password) throws InvalidKeySpecException, IOException {
         final String privateKeyPath = "private.key";
```

Comment:
That is because there is only a single user that is using this app. And this user can only have one private key. So there is no reason to store multiple private keys on a single device for this app.